docs/lib/auth/fragments/android/signin/60_check_sign_in_state.md

```diff
@@ -0,0 +1,55 @@
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+Amplify.Auth.fetchAuthSession(
+    result -> Log.i("AuthQuickstart", result.isSignedIn() ? "User is signed in" : "User is signed out"),
```

Comment:
Per https://github.com/aws-amplify/amplify-android/issues/1247, `result.isSignedIn()` is still true even after the session is expired.  I believe we determined that we need to explicitly try to obtain the token, right and verify that an exception is not thrown right?  Can you add this check?  We also need to make sure our sample app is updated and tested with this approach.